README.md

```diff
@@ -127,7 +127,7 @@ npm run build
 
 # Start server
 ## Note: You need MongoDB running
-npm start
+npm prod
```

Comment:
This won't work. `npm start` and `npm test` work without `run` (possibly others I don't know of) but `prod` requires `npm run prod`